### ISSUE TITLE
[Broodmother] Newborn Fighters

### ISSUE THREAD

**Issue by Doct0rSilver** [QuantumXenon/origins-plus-plus#38]
**Version**
Fabric 0.14.8, mod version 1.6.2

**Origin**
Broodmother

**Power**
Newborn Fighters

**Notes**
The description says you can summon some spiders to fight for you, but they don't target other mobs and just kinda chill out. They're cute and I love them, but they aren't very useful.

**Comment by QuantumXenon** [comment]
I've been working on this, but I can't seem to get it to work - it appears that the spawned spiders are neutral and don't attack anything

**Comment by QuantumXenon** [comment]
There is a spawn event that would do this, but it is only available in Bedrock

**Comment by QuantumXenon** [comment]
As for the `Need Help` tag, if anyone knows how to summon spiders that are already aggressive, that would be very helpful

**Comment by QuantumXenon** [comment]
Closing this as I don't think I will be able to fix this. The power has been removed until there is a way to do this.

**Comment by EvanoriMemoria** [comment]
I recently spent some time working on this issue, I dug deep into the origins discord and found an aggro power that works pretty well, it's not perfect, but good a step forward. I've implemented it into a modified version of the broodmother here: https://github.com/EvanoriMemoria/ReincarnationOrigins/tree/master/src/main/resources/data/reincarnation-origins/powers/broodmother

broodmother/aggro and broodmother/summon_the_army are the two primary files that interact with aggroing your minions, though I added a bit to broodmother/websling too. The primary issue with it is the invincibility frames, if we could get rid of those we would be golden! I also added knockback resist to the minions so the fake damage doesn't interfere with them as much.

It should work even better in newer versions of origins with the Selector Action: https://origins.readthedocs.io/en/latest/types/entity_action_types/selector_action/
